examples/apps/heart_disease_eval/client/UsingHeartEvalGUI.md

```diff
@@ -0,0 +1,17 @@
+# Using the Heart Evaluation GUI Client
+
+1. Follow instructions in the "Testing" section of the [build document](../../BUILD.md) up to step 2
+2. In Terminal 2, run `cd $TCF_HOME/tests/clients`
+3. Open a new terminal, Terminal 3.
+4. In Terminal 3 run `xhost +local:`
+
+   This will give access of your display to the virtual environment.
+5. In Terminal 2, run `python3 heart_gui.py` (use the `--help` option to see other available options)
```

Comment:
When i try to run the heart_gui.py, I am getting error ModuleNotFoundError: No module named 'json_rpc_request'